app/i18n-main.ts

```diff
@@ -0,0 +1,26 @@
+import i18next from 'i18next';
```

Comment:
Just a reminder to drop this file before merging.

Reply:
I didn't want to drop `i18-main.ts` itself just in case a solution was found... but I'll do that anyway.

Reply:
Done